docs/sources/mimir/operators-guide/run-production-environment/scaling-out.md

```diff
@@ -136,7 +136,12 @@ The required amount of time to wait depends on your configuration and it's the m
 
 To guarantee no downtime when scaling down [store-gateways]({{< relref "../architecture/components/store-gateway.md" >}}), complete the following steps:
 
-1. Scale down no more than two store-gateways at the same time.
 1. Ensure at least `-store-gateway.sharding-ring.replication-factor` store-gateway instances are running (three when running Grafana Mimir with the default configuration).
-
-> **Note:** If you enabled [zone-aware replication]({{< relref "../configure/configure-zone-aware-replication.md" >}}) for store-gateways, you can in parallel scale down any number of store-gateway instances in one zone at a time.
+1. Scale down no more than two store-gateways at the same time. If you enabled [zone-aware replication]({{< relref "../configure/configure-zone-aware-replication.md" >}})
```

Comment:
```suggestion
1. If you enabled [zone-aware replication]({{< relref "../configure/configure-zone-aware-replication.md" >}}), scale down no more than two store-gateways at the same time.
```
Rationale: save the reader a tad of time if they do no have zone-aware replication enabled. (They can stop reading and go on to the next step.)